Xamarin.Forms.Platform.UAP/ListViewRenderer.cs

```diff
@@ -137,6 +160,10 @@ void ReloadData()
 
 		void OnCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
 		{
+			/* #fix=7167
+			 * Note: _collectionIsWrapped is only true, if the underlying ItemsSource did not implement IList and INotifyPropertyChanged.
+			 * In that case, a separate _collection is created (ObservableCollection<object>) which is kept in-sync using this method.
+			 */
```

Comment:
Please delete the comment. :)

Reply:
Got already deleted with the last commit. :)